tests/unit/test_scalar.py

```diff
@@ -59,6 +59,16 @@ def test_truediv():
     assert str(field("foo") / 10) == """'foo' / 10"""
 
 
+def test_truediv__literal_right():
+    """Should allow a literal value as the right operand when the left operand is a binary operation"""
+    assert str((field("foo") / 10) * 100) == """('foo' / 10) * 100"""
+
+
+def test_truediv__literal_left():
+    """Should require a literal value as the left operand when the right operand is a binary operation"""
```

Comment:
is this actually being enforced anywhere in code? or does it just naturally raise an exception if you try to do number * non-number?